examples/01_simple.py

```diff
@@ -34,17 +34,17 @@ def cli(debug):
 )
 @click.option("--all", is_flag=True, help="Sync all the things?")
 def sync(type, all):
-    """Synchronise all your files between two places"""
+    """Synchronise all your files between two places."""
     print("Syncing")
 
 
 @cli.command(short_help="Optionally use short-help for the group help text")
 @click.option("--all", is_flag=True, help="Get everything")
 def download(all):
-    """
-    Pretend to download some files from
-    somewhere. Multi-line help strings are unwrapped
-    until you use a double newline.
+    r"""
+    Pretend to download some files from somewhere.
+
+    Multi-line help strings are unwrapped until you use a double newline.
```

Comment:
Intention of this example is to _not_ have a double newline.

```suggestion
    """
    Pretend to download some files from somewhere.
    Multi-line help strings are unwrapped until you use a double newline.
```

Reply:
Changed this back with noqas